packages/forms/resources/views/components/wizard.blade.php

```diff
@@ -160,13 +160,23 @@ class="fi-fo-wizard-header-step-icon-ctn flex h-10 w-10 shrink-0 items-center ju
                                 getStepIndex(step) < {{ $loop->index }},
                         }"
                     >
-                        <x-filament::icon
-                            alias="forms::components.wizard.completed-step"
-                            icon="heroicon-o-check"
-                            x-cloak="x-cloak"
-                            x-show="getStepIndex(step) > {{ $loop->index }}"
-                            class="fi-fo-wizard-header-step-icon h-6 w-6 text-white"
-                        />
+                        @if (filled($completedIcon = $step->getCompletedIcon()))
+                            <x-filament::icon
+                                alias="forms::components.wizard.completed-step"
+                                :icon="$completedIcon"
+                                x-cloak="x-cloak"
+                                x-show="getStepIndex(step) > {{ $loop->index }}"
+                                class="fi-fo-wizard-header-step-icon h-6 w-6 text-white"
+                            />
+                        @else
+                            <x-filament::icon
+                                alias="forms::components.wizard.completed-step"
+                                icon="heroicon-o-check"
+                                x-cloak="x-cloak"
+                                x-show="getStepIndex(step) > {{ $loop->index }}"
+                                class="fi-fo-wizard-header-step-icon h-6 w-6 text-white"
+                            />
+                        @endif
```

Comment:
```suggestion
                        <x-filament::icon
                            :alias="filled($completedIcon) ? null : 'forms::components.wizard.completed-step'"
                            :icon="$completedIcon ?? 'heroicon-o-check'"
                            x-cloak="x-cloak"
                            x-show="getStepIndex(step) > {{ $loop->index }}"
                            class="fi-fo-wizard-header-step-icon h-6 w-6 text-white"
                        />
```

Reply:
Hey @danharrin, Thank you for the update. 

Correct me If am wrong. Should We get `$completedIcon` from the $step first?

Reply:
@danharrin  I will submit a new pull request, Wizard got an execption. `Undefined variable $completedIcon`

Want to make sure why was the alias changes to null on custom?